std/uuid.d

```diff
@@ -256,25 +259,28 @@ public struct UUID
             assert(ctfe.data == data);
         }
 
-/+
-        Not Working! DMD interprets the ubyte literals as ints, then complains the int can't
-        be converted to ubyte!
```

Comment:
no but, I added an template restriction that only allows integrals. Inside the ctor I use the "to" template to convert them to ubyte. So every int type passed in whose value fits into a ubyte will work.